src/main/java/com/github/fairit/destinypro/controller/PlayerController.java

```diff
@@ -1,29 +1,57 @@
 package com.github.fairit.destinypro.controller;
 
+import com.github.fairit.destinypro.dto.character.CharacterData;
 import com.github.fairit.destinypro.dto.character.Characters;
+import com.github.fairit.destinypro.dto.player.api.PlayerApi;
+import com.github.fairit.destinypro.dto.pve_pvp_stats.CharacterPve;
+import com.github.fairit.destinypro.dto.pve_pvp_stats.CharacterPvp;
+import com.github.fairit.destinypro.exception.CharacterNotFoundException;
 import com.github.fairit.destinypro.service.character_service.CharacterService;
 import com.github.fairit.destinypro.service.player_service.PlayerService;
+import com.github.fairit.destinypro.service.stats_service.StatisticService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import static org.springframework.web.bind.annotation.RequestMethod.POST;
-
 @RestController
 public class PlayerController {
 
     private PlayerService playerService;
     private CharacterService characterService;
+    private StatisticService statisticService;
 
     @Autowired
-    public PlayerController(final PlayerService playerService, final CharacterService characterService) {
+    public PlayerController(final PlayerService playerService, final CharacterService characterService, StatisticService statisticService) {
         this.playerService = playerService;
         this.characterService = characterService;
+        this.statisticService = statisticService;
     }
 
-    @RequestMapping(value = "/findplayer/{nickname}", method = POST)
+    @PostMapping(value = "/findplayer/{nickname}")
     public Characters searchPlayerByNicknameAndShowCharacters(@PathVariable String nickname) {
-        return characterService.getListOfPlayerCharacters(playerService.findPlayerByNickname(nickname));
+        PlayerApi playerByNickname = playerService.findPlayerByNickname(nickname);
+        return characterService.getListOfPlayerCharacters(playerByNickname);
+    }
+
+    @PostMapping(value = "/findplayer/{nickname}/pvpstats/{characterId}")
```

Comment:
I don't know how to change the method to accept the nickname from the previous POST. In this method I need only characterID for found player.